### PR TITLE
update user recommendations and simulate data flow through entire service

### PR DESCRIPTION
**Pull request by arsimr16**
https://trello.com/c/Lf20T34C/82-update-recommendations-for-users-from-message-bus-data
https://trello.com/c/rTecTAf7/94-set-up-cron-jobs-to-simulate-data-flow-throughout-app-server

### CODE REVIEW

File: scripts/cronJobs.js [hunk 1]
```diff
@@ -1,15 +1,117 @@
...
+      timestamp: new Date(),
+    });
+  }
+  const sessionData = {
+    userId: Math.floor(Math.random() * 1000000),
+    events,
+  };
+  const options = {
+    url: 'http://localhost:3000/tetraflix/sessionData',
+    headers: {
```
Comment (shwetachari) [inline scripts/cronJobs.js:40]: Is this necessary? In other words, is your server actually checking for this information?

File: scripts/cronJobs.js [hunk 2]
```diff
@@ -1,15 +1,117 @@
...
+    if (error) {
+      throw error;
+    }
+  });
+};
+
+// CRON JOBS
 
 // update genre recommendations every hour
 cron.schedule('* 59 * * * *', genreRecs);
```
Comment (shwetachari) [inline scripts/cronJobs.js:82]: I'm not 100% sure on the syntax, but you might be able to achieve the same timed intervals with '* * *\1 * *'. It's a minimal change, but if you want to make it, you may need to double check whether this is valid!

**Review by shwetachari — APPROVED** [review]
Looks good!

File: scripts/cronJobs.js (hunk 2 above)
Comment (arsimr16) [reply scripts/cronJobs.js:82]: I think that is a valid way of doing it.  There are definitely multiple ways of setting the timing.

File: scripts/cronJobs.js (hunk 1 above)
Comment (arsimr16) [reply scripts/cronJobs.js:40]: I think it is necessary to specify the content-type so that my server knows it is receiving a JSON object.

**Pull request merged**